### PR TITLE
feat: also list immutable keys

### PR DESCRIPTION
**Pull request by fmorency**
Support listing keys marked immutable by their owner.

Relates https://github.com/liftedinit/many-js/pull/112

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "@emotion/react": "^11.10.4",
     "@emotion/styled": "^11.10.4",
     "@fontsource/rubik": "^4.5.11",
-    "@liftedinit/many-js": "github:liftedinit/many-js",
+    "@liftedinit/many-js": "github:fmorency/many-js#update-kvstore-list",
```
Comment (stanleyjones) [inline package.json:10]: Planning to change this before merge?

File: src/api/services/queries/data.ts [hunk 2]
```diff
@@ -9,13 +9,12 @@ import {
 
 export function useListKeys(
   neighborhood: Network | undefined,
-  address: string = ""
+  address: string = "",
+  params: {}[]
```
Comment (stanleyjones) [inline src/api/services/queries/data.ts:13]: I'm surprised this didn't give you a warning. It looks like a required parameter (`params`) after an optional one (`address`).

What exactly is `params` in this case? Can we be more descriptive?

File: src/pages/services/data/data.tsx [hunk 3]
```diff
@@ -1,33 +1,39 @@
...
+  const {isOpen, onOpen, onClose} = useDisclosure();
+  const [keyvalue, setKeyvalue] = useState({key: "", value: ""});
 
-  const { data: keys } = useListKeys(neighborhood, account?.address.toString());
-  const values = useGetValues(neighborhood, keys?.keys);
-  const queries = useQueryValues(neighborhood, keys?.keys);
+  const list = useListKeys(neighborhood, account?.address.toString(),
+    [
+      {filter: [[0, account?.address.toString()], [2, false]]},               // Return all keys that are owned by address and NOT disabled
+      {filter: [[0, "maiyg"], [1, account?.address.toString()], [2, false]]}  // Return all keys that are immutable and previously owned by address and NOT disabled
```
Comment (stanleyjones) [inline src/pages/services/data/data.tsx:25]: Ah, now I see what `params` is for. I guess I'd probably move this logic out of the view layer and into the queries. Something like:

```js
const list = useListKeys(neighborhood, account?.address.toString()));
```

And then:
```js
export function useListKeys (...snip...) {
  return [
    useQuery(...snip...), // currently owned
    useQuery(...snip...), // immutable, previously owned
  ];
}
```

File: package.json (hunk 1 above)
Comment (fmorency) [reply package.json:10]: Yes, I will change it before merging.

File: src/pages/services/data/data.tsx (hunk 3 above)
Comment (fmorency) [reply src/pages/services/data/data.tsx:25]: Sure, can do.

**Review by stanleyjones — APPROVED** [review]
Thanks!

But I think my other PR just introduced conflicts. Sorry!

**Pull request merged**